### ISSUE TITLE
RGB Image Write

### ISSUE THREAD

**Issue by Ajb2k3** [m5stack/UiFlow-Bug-Report#27]
We need a block called RGB Image Write that allows user to display a specifically formatted image on RGB arrays like the neoflash's
We also need a second block called RGB reverse Even row for displays that have the even lines that start on oppersite sides of displays.

**Comment by EeeeBin** [comment]
it`s mean need draw bit map fun?

**Comment by Ajb2k3** [comment]
no, the block loads an image from the M5Stack, and converts it to the 
neopixel0.setColor(1, 0xff0000)

commands needed to illuminate the rbg arrays.
